src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -334,15 +337,13 @@ module.exports = function<T, P, I, TI, C, CX>(
       // Proceed under the assumption that this is a class instance
       workInProgress.tag = ClassComponent;
       adoptClassInstance(workInProgress, value);
-      mountClassInstance(workInProgress, priorityLevel);
```

Comment:
It seems to me that `mountClassInstance` disappeared from this path now. Should be easy to cover this in the test by reading from `props` and `context` as well as using `componentWillMount`.